README.md

```diff
@@ -14,32 +14,38 @@ To get usage information for the `bdk-cli` bin use the below command which retur
 available wallet options and commands:
 
 ```shell
-cargo run --features repl,electrum
+cargo run
 ```
 
 To sync a wallet to the default electrum server:
 
 ```shell
-cargo run --features repl,electrum,esplora -- wallet --descriptor "wpkh(tpubEBr4i6yk5nf5DAaJpsi9N2pPYBeJ7fZ5Z9rmN4977iYLCGco1VyjB9tvvuvYtfZzjD5A8igzgw3HeWeeKFmanHYqksqZXYXGsw5zjnj7KM9/*)" sync
+cargo run -- wallet --descriptor "wpkh(tpubEBr4i6yk5nf5DAaJpsi9N2pPYBeJ7fZ5Z9rmN4977iYLCGco1VyjB9tvvuvYtfZzjD5A8igzgw3HeWeeKFmanHYqksqZXYXGsw5zjnj7KM9/*)" sync
 ```
 
 To get a wallet balance with customized logging:
 
 ```shell
-RUST_LOG=debug,sled=info,rustls=info cargo run --features repl,electrum,esplora -- wallet --descriptor "wpkh(tpubEBr4i6yk5nf5DAaJpsi9N2pPYBeJ7fZ5Z9rmN4977iYLCGco1VyjB9tvvuvYtfZzjD5A8igzgw3HeWeeKFmanHYqksqZXYXGsw5zjnj7KM9/*)" get_balance
+RUST_LOG=debug,sled=info,rustls=info cargo run -- wallet --descriptor "wpkh(tpubEBr4i6yk5nf5DAaJpsi9N2pPYBeJ7fZ5Z9rmN4977iYLCGco1VyjB9tvvuvYtfZzjD5A8igzgw3HeWeeKFmanHYqksqZXYXGsw5zjnj7KM9/*)" get_balance
 ```
 
 To generate a new extended master key, suitable for using in a descriptor:
 
 ```shell
-cargo run --features repl,electrum -- key generate
+cargo run -- key generate
 ```
 
 To install dev version of `bdk-cli` from local git repo:
 
 ```shell
 cd <bdk-cli git repo directory>
-cargo install --path . --features repl,electrum,esplora
+cargo install --path . 
```

Comment:
Sorry, minor nit: there's a whitespace at the end of this line.. I've only noticed it now while trying to merge it from the command line, sorry

Reply:
removed